lm75bd/lm75bd.h

```diff
@@ -5,7 +5,11 @@
 #include <stdint.h>
 
 /* LM75BD I2C Device Address */
-#define LM75BD_OBC_I2C_ADDR /* Define the address here */
+#define LM75BD_OBC_I2C_ADDR  0x4F //1001111 /* Define the address here */
+
+// Temp register address decleration
+
+#define TEMP_REG 0x00 // since it's 000000003
```

Comment:
tbh idk what i was thinking when i wrote that... there's a section of code with comments that are just.. bad because i did them while staying up. ill clean them up :)